spec/models/course/forum_spec.rb

```diff
@@ -24,5 +24,41 @@
         end
       end
     end
+
+    describe '.with_topic_count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.